public/js/main.js

```diff
@@ -39,7 +39,8 @@
     }
 
     kitten_imagery.setAttribute('class', 'kitten')
-
+    kitten_imagery.setAttribute('href', window.kittens[index].url)
```

Comment:
@InfAmelia I have a separate question about this change. For now, can we hold off on linking the videos to any place in particular? Would like to keep the focus of this PR on adding the Giphy logo and have a separate discussion about what we want the kitten GIFs to link to (if anything). <3